### ISSUE TITLE
BUG: getaddrinfo ENOTFOUND

### ISSUE THREAD

**Issue by rmamba** [rmamba/letsproxy#59]
```
Error: getaddrinfo ENOTFOUND www.old.barv*.net www.old.barv*.net:443
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:67:26)
    at Function.module.exports.loopWhile (/home/letsproxy/GIT/letsproxy/node_modules/deasync/index.js:71:23)
    at Object.$wait.use.promise (/home/letsproxy/GIT/letsproxy/node_modules/wait-for-stuff/index.js:153:13)
    at Wget.data (/home/letsproxy/GIT/letsproxy/modules/wget.js:32:29)
    at Acme.writeConfig (/home/letsproxy/GIT/letsproxy/modules/configs/acme.js:49:33)
    at Object.keys.forEach.domain (/home/letsproxy/GIT/letsproxy/modules/configs/acme.js:86:25)
    at Array.forEach (<anonymous>)
    at Acme.writeConfigs (/home/letsproxy/GIT/letsproxy/modules/configs/acme.js:85:31)
    at Letsproxy.writeConfigs (/home/letsproxy/GIT/letsproxy/modules/configs/letsproxy.js:34:15)
    at Letsproxy.writeDomains (/home/letsproxy/GIT/letsproxy/modules/configs/letsproxy.js:270:15)
```

**Comment by rmamba** [comment]
only happens on my dev server?

**Comment by rmamba** [comment]
Additional info: this happens when the IP of the domain is not pointing to letsproxy server

```
Error: getaddrinfo ENOTFOUND letsproxy.domain.com
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:66:26)
    at Function.module.exports.loopWhile (/home/ubuntu/GIT/letsproxy/node_modules/deasync/index.js:71:23)
    at Object.promise (/home/ubuntu/GIT/letsproxy/node_modules/wait-for-stuff/index.js:153:13)
    at Wget.data (/home/ubuntu/GIT/letsproxy/modules/wget.js:32:29)
    at Acme.writeConfig (/home/ubuntu/GIT/letsproxy/modules/configs/acme.js:49:33)
    at /home/ubuntu/GIT/letsproxy/modules/configs/acme.js:88:25
    at Array.forEach (<anonymous>)
    at Acme.writeConfigs (/home/ubuntu/GIT/letsproxy/modules/configs/acme.js:87:31)
    at Letsproxy.writeConfigs (/home/ubuntu/GIT/letsproxy/modules/configs/letsproxy.js:34:15)
    at Letsproxy.writeDomains (/home/ubuntu/GIT/letsproxy/modules/configs/letsproxy.js:272:15)
```

**Comment by rmamba** [comment]
fixed in v0.2.14
